admin_guide/manage_nodes.adoc

```diff
@@ -271,63 +271,68 @@ on selected nodes.
 is critical to ensure that they can remain in operation with one pod
-temporarily going offline.  While implementing resiliency for stateful pods
+temporarily going offline. While implementing resiliency for stateful pods
 is different for each application, in all cases it is important to configure
 the scheduler to use xref:scheduler.adoc#anti-affinity[node anti-affinity] to
-ensure that they pods are properly spread across available nodes.
+ensure that the pods are properly spread across available nodes.
 
 Another challenge is how to handle nodes that are running critical
-infrastructure such as the router or the registry.  The same node evacuation
+infrastructure such as the router or the registry. The same node evacuation
 process applies, though it is important to understand certain edge cases.
 
 [[infrastructure-nodes]]
-=== Infrastructure Nodes 
+=== Infrastructure Nodes
+
 Infrastructure nodes are nodes that are labeled to run pieces of the
-{product-title} environment.  Currently, the easiest way to manage node reboots
+{product-title} environment. Currently, the easiest way to manage node reboots
 is to ensure that there are at least three nodes available to run
-infrastructure.  The scenario below demonstrates a common mistake that can lead
+infrastructure. The scenario below demonstrates a common mistake that can lead
 to service interruptions for the applications running on {product-title} when
 only two nodes are available.
 
-1. Node A is marked unschedulable and all pods are evacuated
-1. The registry pod running on that node is now redeployed on node B.  This
-   means node B is now running both registry pods.
-1. Node B is now marked unschedulable and is evacuated
-1. The service exposing the two pod endpoints on node B, for a brief period of
-   time, looses all endpoints until they are redeployed to node A
-
-The same proces using three infrastructure nodes will not result in a service
-disruption.  However, due to pod scheduling, the end result is one node
-running two registry pods, one running one, and another running zero.  The best
-solution is to rely on pod anti-affinity.  This is an alpha feature in
-kubernetes that is available for testing now, but is not yet supported for
-production workloads.
+- Node A is marked unschedulable and all pods are evacuated.
+- The registry pod running on that node is now redeployed on node B. This means
+node B is now running both registry pods.
+- Node B is now marked unschedulable and is evacuated.
+- The service exposing the two pod endpoints on node B, for a brief period of
+   time, loses all endpoints until they are redeployed to node A.
+
+The same process using three infrastructure nodes does not result in a service
+disruption. However, due to pod scheduling, the end result is one node running
+two registry pods, the node you restarted with zero registries, and the node you
+have not touched yet with zero registries. The last node you restart will have
```

Comment:
The node that hasn't been touched will have 1 registry.

This situation is hard to explain, but it seems a little wordy to me.